Source/Particles/ElementaryProcess/QEDSchwingerProcess.H

```diff
@@ -65,9 +69,9 @@ struct SchwingerTransformFunc
      * @param[in] total_weight Number of physical pairs created in the
      *            considered cell.
      */
-    template <typename DstData1, typename DstData2>
+    template <typename DstTile>
     AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
-    void operator() (DstData1& dst1, DstData2& dst2, const int i_dst1,
+    void operator() (DstTile& dst1, DstTile& dst2, const int i_dst1,
```

Comment:
Is there a particular reason for having `dst1` and `dst2` of the same type here?

Reply:
Not really, it was just to remain consistent with `FilterCreateTransformFromFAB` and `FilterCopyTransform` which both use the same template parameters for the type of dst1 and dst2. I can change that if you think it's better.